core/templates/utility/forms.ts

```diff
@@ -0,0 +1 @@
+export const VALIDATION_STATUS_INVALID = 'INVALID';
```

Comment:
Please include the usual stuff at the top of this TS file.

Also, in an earlier comment, you mentioned "Adding a comment about this in the code might distract new validations from using this." I don't see a comment warning devs against using this?

Reply:
Done

Reply:
(is it safe for other devs to use it.)